definitions/structs/embed.yml

```diff
@@ -1,6 +1,15 @@
 ---
 name: Embed
-description: An embed in a message, such as for a URL or an attachment.
+description: >
+  Represents a rich embed which allows using richer markdown,
+  multiple columns and more. This was heavily inspired by [slack's
+  attachments](https://api.slack.com/docs/message-attachments).
+
+  You can include an attachment in your own message by a user or a bot,
+  or in a webhook. They can be edited, too.
+
+  **Note**: Maximum amount of characters in a field name is 256,
+  in field value - 1024 and in description - 2048
```

Comment:
We should also document this in the description/name/value methods in [CreateEmbed] and [CreateEmbedField].

[CreateEmbed]: https://serenity.zey.moe/serenity/utils/builder/struct.CreateEmbed.html
[CreateEmbedField]: https://serenity.zey.moe/serenity/utils/builder/struct.CreateEmbedField.html